modnet/preprocessing.py

```diff
@@ -178,8 +162,8 @@ def get_c(nn):
         logging.debug("In selection of feature {}/{} features...".format(n+1, n_feat))
         if (n+1) % 50 == 0:
             logging.info("Selected {}/{} features...".format(n, n_feat))
-        p = get_p(nn=n)
-        c = get_c(nn=n)
+        p = get_p(n)
+        c = get_c(n)
```

Comment:
I don't know if you wanna look at what I've done in the other PR to use as defaults? Either way, we should think about reviewing and merging what I've done, if you're okay with it...

https://github.com/ppdebreuck/modnet/pull/7/files#diff-c89c000a5346b2c316d732e6acc04badR173-R184

Reply:
I will definitely take a look and merge it, thought it was still in progress, sorry. For the rr params, putting c and p as argument is indeed not a good solution, as we will use other type of functions in the future...
We should discuss on some intuitive dictionary definition. The current dictionary is quite confusing with ['p']['function'] == 'constant' not needed.